backend/backend/player.py

```diff
@@ -81,11 +81,12 @@ def __exit__(self, *exc):
                                (self.username, self.balance,
                                 self.turn_position, self.board_position,
                                 self.uid))
-                cursor.executemany('REPLACE INTO `rolls` '
-                                   'VALUES (%s, %s, %s, %s);',
-                                   ((self.uid, roll1, roll2, i)
-                                    for i, (roll1, roll2)
-                                    in enumerate(self.rolls)))
+                if self.rolls != []:
```

Comment:
Why might I ask is this if-statement here? Is it just to save the database from needlessly executing SQL if self.rolls is empty

Reply:
If this executes before any rolls have happened it crashes the exit.

Reply:
Thanks for letting me know, flip that's important!